Source/Python/utils/mp4-dash.py

```diff
@@ -494,6 +495,9 @@ def OutputDash(options, set_attributes, audio_sets, video_sets, subtitles_sets,
             language = audio_tracks[0].language
             if (language != 'und') or options.always_output_lang:
                 kwargs['lang'] = language
+            language_name = audio_tracks[0].language_name
+            if (language_name != 'Unknown'):
+              kwargs['lang'] = language_name
```

Comment:
This isn't going to work. The @lang attribute of an <AdaptationSet> element is the language code, not the language name (see ISO/IEC 23009-1, section 5.3.3.2): "@lang: Declares the language code for this Adaptation Set. The syntax and semantics according to IETF RFC 5646 shall be used."

Reply:
You're right, the `@lang` value shall be RFC 5646 language code, my mistake.

What do you think about using the "label" attribute? Although it is non-standard (as far as i know, i don't have access to ISO/IEC 23009, so i can't say for sure), support for "label" was implemented in some major dash players (google/ExoPlayer#4391, google/shaka-player#811)

Reply:
I have just pushed an update that allows setting track labels (use the `+label=xxx` track option. This way you pass an explicit label (setting the label to be equal to the language name doesn't help with dash, since a player would normally already do that mapping from the language code, so the explicit labelling is only interesting when you want a label that's not the language name, or at least a different language name than the standard name for a language code)

Reply:
Makes sense, i close the pr

Reply:
is it possible to use same way for the subtitles ?